stories/Chessboard.stories.tsx

```diff
@@ -511,6 +511,7 @@ export const PremovesEnabled = () => {
           boxShadow: "0 2px 10px rgba(0, 0, 0, 0.5)",
         }}
         ref={chessboardRef}
+        dragOutsideOfBoard={false}
```

Comment:
looks like it's still referenced as  dragOutsideOfBoard in a lot of places instead of the new name, maybe do a find and replace on this value,

can you also add this prop to the README

Reply:
This is why I shouldnt do it on my lunch break. I'll update it everywhere hopefully tonight after work.

Reply:
@Clariity Fixed for real this time